massgov/mayflower
# DP-12682/DP-12682 How to page template mapping

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

Some changes were needed on the Mayflower side to allow the template mapping of the How-to page to work correctly in Drupal. Removed the How-to social media links from the How-to page and added a block to the tabular-data.twig.

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-12682)
- [Github issue]()

## Steps to Test


1. Compare the How to page `http://localhost:3000/?p=pages-howto` vs `https://mayflower.digital.mass.gov/?p=pages-howto`
  1. The following text was changed: `Next Steps` to `Next steps` and `How To Apply` to `How to apply` to match the www.mass.gov styleguide.
  1. Social media links in the header should be the only item that has been removed from the Howto page. Everything else should be the same.
1.  Compare the tabular-data.twig `http://localhost:3000/?p=organisms-tabular-data` vs `https://mayflower.digital.mass.gov/?p=organisms-tabular-data`. This should not have any changes to this organism with the block being added.
 
## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.
Howto page removed these social media links:
<img width="1308" alt="howto page change" src="https://user-images.githubusercontent.com/19477691/53928186-3494a500-4057-11e9-8994-2b09e9c85078.png">



## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
